brave/brave-ios
# Fix #4377: Revamp 3dot menu on new tab page

## Summary of Changes


This pull request fixes #4377

## Submitter Checklist:


## Test Plan:

- Check Options menu is not updated for loaded page setup
- Check the items exist only in NTP with proper description
- Check Brave News / Brave Talk not in menu for Private Browsing
- Check New only shown when Brave news is enabled
- Check News scrolls down the news section
- Also check talk opens the website

## Screenshots:



https://user-images.githubusercontent.com/6643505/139927730-05b5c2ef-dd15-4215-b6fe-6856199c727c.MP4




## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `bug` / `enhancement`
